Sources/SkeletonView.swift

```diff
@@ -57,6 +73,15 @@ extension UIView {
         recursiveShowSkeleton(withType: type, usingColors: colors, animated: animated, animation: animation)
     }
     
+    func updateSkeleton(withType type: SkeletonType = .solid, usingColors colors: [UIColor], animated: Bool = false, animation: SkeletonLayerAnimation? = nil) {
+        skeletonIsAnimated = animated
+        if flowDelegate == nil {
+            flowDelegate = SkeletonFlowHandler()
+            flowDelegate?.willBeginShowingSkeletons(withRootView: self)
```

Comment:
I was thinking about using an update notification too but was not sure about it. So I've left the begin notification only as you can call `updateSkeleton` instead of `showSkeleton` and will be the same as calling `showSkeleton`.

I think the best solution will be to notify `begin` if neither `updateSkeleton` or `showSkeleton` has been called, and notify always about an `update`. I'll update it this way.

Reply:
I've thought about this, and I'm not sure if we should check if flowDelegate is != nil, I mean, we are updating the skeleton, so flowDelegate should be != nil, if not something was wrong, but this is another problem. What do you think?

Reply:
I think it's necessary to create `flowDelegate` in case it's nil as the `updateOrShowSkeleton` method acts as a show or update function. If it's called before any `showSkeleton` call, it acts as the same as calling `showSkeleton`. If the `updateOrShowSkeleton` method is called after a `showSkeleton` call, then it only updates the layers.

I've updated the method name from `updateSkeleton` to `updateOrShowSkeleton` so now it reflects that it shows or updates the skeleton layers.